runtime/opt/taupage/init.d/10-prepare-disks.py

```diff
@@ -372,17 +386,17 @@ def main():
     else:
         configure_logging(logging.INFO)
 
+    global current_region
```

Comment:
Why do we need this? Using `global` smells a bit to me, but I'm not really a python expert.